apps/happy-blocks/src/pricing-plans/components/pricing-plans-tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import classnames from 'classnames';
+import { FunctionComponent } from 'react';
+import usePlanOptions from '../hooks/plan-options';
+import { BlockPlan } from '../hooks/pricing-plans';
+import { BlockAttributes } from '../types';
+
+interface Props {
+	currentPlan: BlockPlan;
+	plans: BlockPlan[];
+	setPlan: ( productSlug: string ) => void;
+	attributes: BlockAttributes;
+}
+
+const PricingPlansTabs: FunctionComponent< Props > = ( {
+	currentPlan,
+	attributes,
+	plans,
+	setPlan,
+} ) => {
+	const planOptions = usePlanOptions( plans );
+
+	const availablePlanOptions = planOptions.filter( ( { value } ) =>
+		attributes.planTypeOptions.includes( value )
+	);
+
+	const onTabButtonClick = ( type: string ) => {
+		const plan = plans.find( ( plan ) => plan.type === type && currentPlan.term === plan.term );
+		if ( plan ) {
+			setPlan( plan.productSlug );
+		}
+	};
+
+	return (
+		<section className="hb-pricing-plans-embed__tabs">
+			{ availablePlanOptions.map( ( planOption ) => (
+				<button
+					onClick={ () => onTabButtonClick( planOption.value ) }
+					className={ classnames( {
+						'hb-pricing-plans-embed__tabs-label': true,
```

Comment:
@msurdi-a8c Seems the `.hb-pricing-plans-embed__tabs-label` is must. We can just:

```jsx
classnames( 'hb-pricing-plans-embed__tabs-label', {  ... } );
```